code/game/gamemodes/cult/runes.dm

```diff
@@ -1015,7 +1015,7 @@ var/list/sacrificed = list()
 
 /obj/effect/rune/proc/runestun(var/mob/living/user, var/mob/living/T as mob)
 	if(istype(src,/obj/effect/rune))   ///When invoked as rune, flash and stun everyone around.
-		user.say("Fuu ma[pick("'","`")]jin!")

```

Comment:
What is this a reference to?

Reply:
Not sure, but it felt out of place, and like something from an anime, it doesn't have the consonant abuse of some of the other stuff, so I changed it on the assumption that it was similarly a reference, just like the dream sign nonsense.

Look, this is going to be a full rework of the cult mechanics. If you're going to be huffy about every word changed because it's different now then I don't think you understand what I'm trying to do.  Even if this isn't an anime reference, changing one nonsense phrase to another that sounds better does not affect anything except nostalgia.

Reply:
.. sorry about my tone there, though. I might be getting overly grouchy today.

Reply:
More of a curiosity, if you want some edumacations, Majin is the japanese word for Demon, buuut... I highly doubt it's a reference.

Reply:
Having been told that, a quick search told me that Majin Fuu is a Dragon Ball Z villain.

So pretty sure it's a reference.

Reply:
From Xenoverse 2. Which came out... AHEM. TWO (or fewer I don't care to check) YEARS AGO.

Yes. Xenoverse 2 apparently predates the addition of the cult roundtype. TIL.

Reply:
Ancient memes, ancient sins